File: src/Vincent OS/TaskVOS.vb
Public Class TaskVOS

    Private Sub TaskVOS_Load(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles MyBase.Load
        '' Application
        If Bing_Chrome.Visible = True Then
            PictureBox1.Visible = True
            RadioButton2.Visible = True
        End If
        If Calc.Visible = True Then
            PictureBox2.Visible = True
            RadioButton12.Visible = True
        End If
        If chrono.Visible = True Then
            PictureBox3.Visible = True
            RadioButton20.Visible = True
        End If
        If exewin.Visible = True Then
            PictureBox4.Visible = True
            RadioButton10.Visible = True
        End If
        If game.Visible = True Then
            PictureBox5.Visible = True
            RadioButton8.Visible = True
        End If
        If meme.Visible = True Then
            PictureBox8.Visible = True
            RadioButton3.Visible = True
        End If
        If Notepad.Visible = True Then
            PictureBox9.Visible = True
            RadioButton4.Visible = True
        End If
        If MazeGame.Visible = True Then
            PictureBox6.Visible = True
            RadioButton9.Visible = True
        End If
        If Media.Visible = True Then
            PictureBox7.Visible = True
            RadioButton5.Visible = True
        End If
        If paints.Visible = True Then
            PictureBox10.Visible = True
            RadioButton7.Visible = True
        End If
        If Store.Visible = True Then
            PictureBox11.Visible = True
            RadioButton19.Visible = True
        End If
        If Terminal.Visible = True Then
            PictureBox13.Visible = True
            RadioButton11.Visible = True
        End If
        If Visio.Visible = True Then
            PictureBox14.Visible = True
            RadioButton6.Visible = True
        End If

        '' Service

        If Applications.Visible = True Then
            PictureBox15.Visible = True
            RadioButton15.Visible = True
        End If
        If Bureau.Visible = True Then
            PictureBox16.Visible = True
            RadioButton13.Visible = True
            PictureBox19.Visible = True
            RadioButton17.Visible = True
        End If
        If Bureau2.Visible = True Then
            PictureBox17.Visible = True
            RadioButton14.Visible = True
            PictureBox20.Visible = True
            RadioButton18.Visible = True
        End If
        If Paramètres.Visible = True Then
            PictureBox18.Visible = True
            RadioButton16.Visible = True
        End If
    End Sub

    Private Sub Button1_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles Button1.Click
        If RadioButton2.Checked = True Then
            Bing_Chrome.Close()
        ElseIf RadioButton3.Checked = True Then
            meme.Close()
        ElseIf RadioButton4.Checked = True Then
            Notepad.Close()
        ElseIf RadioButton5.Checked = True Then
            Media.Close()
        ElseIf RadioButton6.Checked = True Then
            Visio.Close()
        ElseIf RadioButton7.Checked = True Then
            paints.Close()
        ElseIf RadioButton8.Checked = True Then
            game.Close()
        ElseIf RadioButton9.Checked = True Then
            MazeGame.Close()
        ElseIf RadioButton10.Checked = True Then
            exewin.Close()
        ElseIf RadioButton11.Checked = True Then
            Terminal.Close()
        ElseIf RadioButton12.Checked = True Then
            Calc.Close()
        ElseIf RadioButton19.Checked = True Then
            Store.Close()
        ElseIf RadioButton20.Checked = True Then
            chrono.Close()
        Else
            Dim backmusic As New System.Media.SoundPlayer(My.Resources.errorSound)
        backmusic.Play()
        MessageBox.Show("Erreur : Vous n'avez sélectionné aucune application à arrêter.", "Erreur", MessageBoxButtons.OK, MessageBoxIcon.Error)
        End If
    End Sub

    Private Sub Button2_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles Button2.Click
        If RadioButton17.Checked = True Then
            Bureau.Close()
            Login.Show()
            Me.Close()
        ElseIf RadioButton18.Checked = True Then
            Bureau2.Close()
            Login.Show()
            Me.Close()
        Else
            Dim backmusic As New System.Media.SoundPlayer(My.Resources.errorSound)
            backmusic.Play()
            MessageBox.Show("Erreur : Vous avez sélectionné aucun utilisateur à arrêter.", "Erreur", MessageBoxButtons.OK, MessageBoxIcon.Error)
        End If
    End Sub

    Private Sub Button3_Click(sender As Object, e As EventArgs) Handles Button3.Click
        If RadioButton13.Checked = True Then
            Bureau.Close()
        ElseIf RadioButton14.Checked = True Then
            Bureau2.Close()
        ElseIf RadioButton15.Checked = True Then
            Applications.Close()
        ElseIf RadioButton16.Checked = True Then
            Paramètres.Close()
        Else
            Dim backmusic As New System.Media.SoundPlayer(My.Resources.errorSound)
            backmusic.Play()
            MessageBox.Show("Erreur : Vous avez sélectionné aucun service à arrêter.", "Erreur", MessageBoxButtons.OK, MessageBoxIcon.Error)
        End If
    End Sub
End Class

File: src/Vincent OS/TaskVOS.cs
using System;
using System.Windows.Forms;

namespace Vincent_OS
{
    public partial class TaskVOS
    {
        public TaskVOS()
        {
            InitializeComponent();
        }

        private void TaskVOS_Load(object sender, EventArgs e)
        {
            Bureau bureau = new Bureau();
            Bureau bureau2 = new Bureau();
            WPF.Bing_Chrome bingChrome = new WPF.Bing_Chrome();
            WPF.Media media = new WPF.Media();
            Applications applications = new Applications();
            Calc calc = new Calc();
            exewin exewin = new exewin();
            game game = new game();
            MazeGame MazeGame = new MazeGame();
            Meme meme = new Meme();
            Notepad Notepad = new Notepad();
            Paramètres Paramètres = new Paramètres();
            Paints paints = new Paints();
            Terminal Terminal = new Terminal();
            Visio Visio = new Visio();
            #region Application
            if (bingChrome.Visibility == System.Windows.Visibility.Visible)
            {
                panel1.Visible = true;
            }
            if (calc.Visible == true)
            {
                panel2.Visible = true;
            }
            if (exewin.Visible == true)
            {
                panel4.Visible = true;
            }
            if (game.Visible == true)
            {
                panel5.Visible = true;
            }
            if (meme.Visible == true)
            {
                panel8.Visible = true;
            }
            if (Notepad.Visible == true)
            {
                panel9.Visible = true;
            }
            if (MazeGame.Visible == true)
            {
                panel6.Visible = true;
            }
            if (media.Visibility == System.Windows.Visibility.Visible)
            {
                panel7.Visible = true;
            }
            if (paints.Visible == true)
            {
                panel10.Visible = true;
            }
            if (Terminal.Visible == true)
            {
                panel12.Visible = true;
            }
            if (Visio.Visible == true)
            {
                panel13.Visible = true;
            }
            #endregion
            #region Services

            if (applications.Visible == true)
            {
                panel14.Visible = true;
            }
            if (bureau.Visible == true)
            {
                panel15.Visible = true;
            }
            if (bureau2.Visible == true)
            {
                panel16.Visible = true;
            }
            if (Paramètres.Visible == true)
            {
                panel17.Visible = true;
            }
            #endregion
            #region Session

            if (bureau.Visible == true)
            {
                PictureBox19.Visible = true;
                RadioButton17.Visible = true;
            }
            if (bureau2.Visible == true)
            {
                PictureBox20.Visible = true;
                RadioButton18.Visible = true;
            }
            #endregion
        }

        #region Applications
        private void Button1_Click(object sender, EventArgs e)
        {
            WPF.Bing_Chrome bingChrome = new WPF.Bing_Chrome();
            WPF.Media media = new WPF.Media();
            exewin exewin = new exewin();
            game game = new game();
            MazeGame MazeGame = new MazeGame();
            Meme meme = new Meme();
            Notepad Notepad = new Notepad();
            Paints paints = new Paints();
            Terminal Terminal = new Terminal();
            Visio Visio = new Visio();
            if (RadioButton2.Checked == true)
            {
                bingChrome.Close();
            }
            else if (RadioButton3.Checked == true)
            {
                meme.Close();
            }
            else if (RadioButton4.Checked == true)
            {
                Notepad.Close();
            }
            else if (RadioButton5.Checked == true)
            {
                media.Close();
            }
            else if (RadioButton6.Checked == true)
            {
                Visio.Close();
            }
            else if (RadioButton7.Checked == true)
            {
                paints.Close();
            }
            else if (RadioButton8.Checked == true)
            {
                game.Close();
            }
            else if (RadioButton9.Checked == true)
            {
                MazeGame.Close();
            }
            else if (RadioButton10.Checked == true)
            {
                exewin.Close();
            }
            else if (RadioButton11.Checked == true)
            {
                Terminal.Close();
            }
            else
            {
                var backmusic = new System.Media.SoundPlayer(My.Resources.Resources.errorSound);
                backmusic.Play();
                MessageBox.Show("Erreur : Vous n'avez sélectionné aucune application à arrêter.", "Erreur", MessageBoxButtons.OK, MessageBoxIcon.Error);
            }
            GC.Collect();
            GC.WaitForPendingFinalizers();
        }
        #endregion
        #region Session
        private void Button2_Click(object sender, EventArgs e)
        {
            Bureau bureau = new Bureau();
            Bureau bureau2 = new Bureau();
            Login login = new Login();
            if (RadioButton17.Checked == true)
            {
                bureau.Close();
                login.Show();
                Close();
            }
            else if (RadioButton18.Checked == true)
            {
                bureau2.Close();
                login.Show();
                Close();
            }
            else
            {
                var backmusic = new System.Media.SoundPlayer(My.Resources.Resources.errorSound);
                backmusic.Play();
                MessageBox.Show("Erreur : Vous avez sélectionné aucun utilisateur à arrêter.", "Erreur", MessageBoxButtons.OK, MessageBoxIcon.Error);
            }
        }
        #endregion
        #region Services
        private void Button3_Click(object sender, EventArgs e)
        {
            Bureau bureau = new Bureau();
            Bureau bureau2 = new Bureau();
            Applications applications = new Applications();
            Paramètres Paramètres = new Paramètres();
            if (RadioButton13.Checked == true)
            {
                bureau.Close();
            }
            else if (RadioButton14.Checked == true)
            {
                bureau2.Close();
            }
            else if (RadioButton15.Checked == true)
            {
                applications.Close();
            }
            else if (RadioButton16.Checked == true)
            {
                Paramètres.Close();
            }
            else
            {
                var backmusic = new System.Media.SoundPlayer(My.Resources.Resources.errorSound);
                backmusic.Play();
                MessageBox.Show("Erreur : Vous avez sélectionné aucun service à arrêter.", "Erreur", MessageBoxButtons.OK, MessageBoxIcon.Error);
            }
        }
        #endregion
    }
}